modules/jupyter-widget/src/plugin.js

```diff
@@ -0,0 +1,29 @@
+// import {Application, IPlugin} from '@phosphor/application';
+
+import {IJupyterWidgetRegistry} from '@jupyter-widgets/base';
+
+import * as widgetExports from './widget';
+
+import {MODULE_NAME, MODULE_VERSION} from './version';
+
+const EXTENSION_ID = 'deckgl-widget:plugin';
+
+const DeckGLPlugin = {
+  id: EXTENSION_ID,
+  requires: [IJupyterWidgetRegistry],
+  activate: activateWidgetExtension,
+  autoStart: true
+};
+
+export default DeckGLPlugin;
+
+/**
+ * Activate the widget extension.
```

Comment:
This comment doesn't add much beyond what the function name says. Maybe some additional context?